testserver/static/src/css/testserver.css

```diff
@@ -1,21 +1,21 @@
 @charset "utf-8";
 .openerp .customer_asset_kanban_left {
-  display: inline-block;
+    display: inline-block;
 }
 
 .openerp .customer_asset_type_kanban_left {
-  display: inline-block;
+    display: inline-block;
 }
 
-#oe_main_menu_navbar.navbar {
-  background-color: #040 !important;
+#oe_main_menu_navbar.navbar-inverse {
+    background-color: #040 !important;
 }
 
-.oe_logo {
-  background-image: url('/testserver/static/src/img/testserver.jpg') !important;
-  background-size: cover !important;
+.o_sub_menu_logo {
+    background-image: url('/testserver/static/src/img/testserver.jpg') !important;
+    background-size: cover !important;
```

Comment:
Could you change "cover" to "contain" ? So the text "TESTSERVER" is always visible fully

Reply:
fixed